### PR TITLE
feat: allow overriding global vars

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments

This allows overriding global vars

## Pull Request Checklist

Thank you for submitting a contribution to our project!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [X] Rebase your branch against the latest commit of the target branch.
- [X] If you are adding a dependency, please explain how it was chosen.
- [X] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results.
- [X] If there is an issue associated with your Pull Request, link the issue to the PR.
- [X] Validate that documentation is accurate and aligned to any project updates or additions.

Don't forget our more detailed contribution guidelines
[here](https://github.com/SeisoLLC/operations/blob/main/documents/software-guidelines.md#contributing-to-a-repository).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
[the remark this answers is not in the document]
Comment (JonZeolla) [reply Dockerfile:2]: No it doesn't 🤔

[the remark this answers is not in the document]
Comment (JonZeolla) [reply Dockerfile:2]: Not a bad idea, but not adding that in this PR.

**Review by seanctech — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (JonZeolla) [reply Dockerfile:2]: Untrue

**Pull request merged**